WebRoot/js/zimbraAdmin/search/model/ZaSearch.js

```diff
@@ -146,8 +146,14 @@ ZaSearch.searchDirectory = function(params) {
         ZaSearch._currentQuery = "";
     }
 
+    var customResultValue;
+    if(params.isInitialRequest) {
+        var multiplier = 1 + parseInt((document.documentElement.clientHeight - 100)/1000, 0);
```

Comment:
* Is `document.documentElement.clientHeight` a string? If it is a number, `parseInt` is unnecessary.
* If parseInt is necessary, why is radix (second param) `0`, not `10` ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt
```
Syntax
parseInt(string)
parseInt(string, radix)
```

Reply:
good catch it should be 10